maintainers/maintainer-list.nix

```diff
@@ -3576,6 +3576,18 @@
     githubId = 1118859;
     name = "Scott Worley";
   };
+  ChlorideCull = {
```

Comment:
Wait so you didn't add this person to the maintainers of the package, why are you adding this entry here?

Reply:
my guess is its just rebase weirdness & coincidence that he's right next to ChlorideCull who removed themselves: https://github.com/NixOS/nixpkgs/pull/307526

Reply:
Something happened when I merged master branch. I will rebase instead to fix up any strangeness.

Reply:
maintainers list is fixed and branch is up-to-date